openshift_images/configuring-samples-operator.adoc

```diff
@@ -8,6 +8,21 @@ toc::[]
 
 The Cluster Samples Operator, which operates in the `openshift` namespace, installs and updates the {op-system-base-full}-based {product-title} image streams and {product-title} templates.
 
+[IMPORTANT]
+.Cluster Samples Operator is being downsized  
+====
+. No RFEs will be accepted for whole of samples operator
+
+. For Non-s2i(Non-Builder Imagestreams/Templates)  
+- No new Imagestreams /Templates 
+- No updates to the existing Imagestreams/templates
+- Continue to be supported as per link:https://access.redhat.com/support/policy/updates/openshift#dates[OCP support guidelines] .
+
+. S2i builder Imagestreams and templates will continue to be supported and updates will be accepted.
```

Comment:
Is there a way to show them the samples that fall into this category?

Reply:
added